streams/src/main/java/org/apache/kafka/streams/kstream/ValueTransformer.java

```diff
@@ -79,7 +79,7 @@
      * @param value the value to be transformed
```

Comment:
Also , the 1-liner description "Transform the given value to a new value." needs updating.